### DISCUSSION TITLE
error saat menggunakan explain() dengan LIME | concrete prediction capstone

### DISCUSSION THREAD

**Discussion by isal-amir** [teamalgoritma/community#486] [Q&A] [answered]
saya mendapatkan error ini saat melakukan explain() dengan package LIME
![image](https://user-images.githubusercontent.com/79739081/207749019-63b5f88d-a369-4171-b56e-ee3f6559325a.png)
awalnya saya menggunakan n_features = 5 karena error saya coba dengan 1 masih error. sepertinya bukan jumlah feature yang menjadi masalah. saya sudah coba cari di google masih belum menemukan jawaban. 
Mohon bantuannya.
Terima kasih sebelumnya.

**Comment by nisa-basalamah** [comment]
Dear @isal-amir ,

Terima kasih atas pertanyaan yang disampaikan, pertanyaan Anda sudah kami terima. Kami akan mencoba membantu mencarikan referensi berkaitan dengan pertanyaan yang diberikan.

**Comment by kinantytasyao** [comment, marked as answer]
Dear @isal-amir 

selected_data yang digunakan pada fungsi `explain()` dan berisi data_test perlu dicek apakah kolomnya sudah sesuai dengan data_train yang dimasukkan ke dalam fungsi `lime()` yang dinamai explainer, selain itu n_features seharusnya diisi dengan banyaknya jumlah variabel prediktor yang digunakan pada model yang digunakan.

Demikian yang dapat kami bantu, semoga bermanfaat :)

**Reply by isal-amir** [reply]
Wah iya betul Kak, terima kasih ternyata di data test saya masih mengandung kolom id. Setelah saya hilangkan bisa berjalan dengan baik.
Terima kasih Kak Kinan.

**Reply by kinantytasyao** [reply]
Sama sama :)
